fdbserver/Status.actor.cpp

```diff
@@ -421,6 +434,28 @@ struct RolesInfo {
 			obj["mutation_bytes"] = StatusCounter(metrics.getValue("MutationBytes")).getStatus();
 			obj["mutations"] = StatusCounter(metrics.getValue("Mutations")).getStatus();
 
+			std::string latencyBandPrefix = "ReadLatency";
+
+			JsonBuilderObject latency;
+			std::map<std::string, JsonBuilderObject> bands;
+
+			bool found = false;
+			for(auto itr = metrics.begin(); itr != metrics.end(); ++itr) {
+				if(itr->first.substr(0, latencyBandPrefix.size()) == latencyBandPrefix) {
+					found = true;
+					std::string band = itr->first.substr(latencyBandPrefix.size());
+					latency[band] = StatusCounter(itr->second).getCounter();
```

Comment:
just use the value?

Reply:
See comment below about decision to keep counters.